File: lib/core/stroop_color.dart
import 'dart:ui';

enum StroopColor {
  red,
  green,
  blue,
  yellow;

  String get displayName => name.toUpperCase();

  String get audioAsset => '$name.mp3';

  Color get flameColor => switch (this) {
        StroopColor.red    => const Color(0xFFFF2D2D),
        StroopColor.green  => const Color(0xFF39FF14),
        StroopColor.blue   => const Color(0xFF1F8FFF),
        StroopColor.yellow => const Color(0xFFFFE600),
      };
}

File: app/src/main/kotlin/com/stroopoverload/core/StroopColor.kt
package com.stroopoverload.core

import androidx.compose.ui.graphics.Color

enum class StroopColor(
    val displayName: String,
    val audioRes: String,
    val composeColor: Color,
) {
    RED("RED", "red", Color(0xFFFF2D2D)),
    GREEN("GREEN", "green", Color(0xFF39FF14)),
    BLUE("BLUE", "blue", Color(0xFF1F8FFF)),
    YELLOW("YELLOW", "yellow", Color(0xFFFFE600));
}
